app/config.py

```diff
@@ -4,6 +4,8 @@
 
 
 class Config():
+    HOST_URL = "http://localhost:6017"
```

Comment:
I think it makes more sense to get this from the environment and not default it in the base class at all.  Perhaps default it to this in the development version